lib/tpl/dokuwiki/style.ini

```diff
@@ -31,7 +31,7 @@ css/_diff.css             = screen
 css/_edit.css             = screen
 css/_modal.css            = screen
 css/_forms.css            = screen
-css/_admin.css            = screen
+css/_admin.less           = screen
```

Comment:
Renaming this would break a few templates which import the file directly in their style.ini. Not the end of the world, but just something we need to consider.